Revision: network-manager--MAIN--0--patch-321
Archive: network-manager@bazaar.ubuntu.com
Creator: Arch Librarian <arch@canonical.com>
Date: Sat Aug 13 15:58:50 BST 2005
Standard-date: 2005-08-13 14:58:50 GMT
Modified-files: ChangeLog
New-patches: network-manager@bazaar.ubuntu.com/network-manager--MAIN--0--patch-321
Summary: *** empty log message ***
CSCVSID: MAIN.322
Keywords: cscvs:MAIN.322

Author: walters
Date: 2004-12-01 22:34:42 GMT
*** empty log message ***
